syntax = "proto2";

import "MessageResponseRoundPlayerBang.proto";
import "MessageResponseRoundPlayerDuelist.proto";

option optimize_for = LITE_RUNTIME;
option java_package = "ProtoMessages";
option java_outer_classname = "MessageResponseTimeToResultsBase";

message MessageResponseTimeToResults {
    enum eRoomResultType{
        rrtPlayerBang = 0;
        rrtPlayerDuelist = 1;
    }
    required int64 roomId = 1 [default = 0];
    required int32 time = 2 [default = 0];
    required eRoomResultType resultType = 3 [default = rrtPlayerBang];
    repeated MessageResponseRoundPlayerBang gameResults = 4;
    repeated MessageResponseRoundPlayerDuelist duelistResults = 5;
}